### PR TITLE
MC-12985: Document get and list network policy rules

### PR DESCRIPTION
**Pull request by patrickspensieri**
#### Changes made

- Document GET and LIST network policy rules

#### Related PRs
- https://github.com/cloudops/cloudmc-stackpath-plugin/pull/24

### CODE REVIEW

File: source/includes/stackpath/_network_policy_rules.md [hunk 1]
```diff
@@ -3,6 +3,114 @@
 Network policy rules allows you to control inbound and outbound traffic to your [workload](#stackpath-workloads).
 
 
+<!-------------------- LIST NETWORK POLICY RULES -------------------->
+### List network policy rules
+
+```shell
+curl -X GET \
+  -H "MC-Api-Key: your_api_key" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules"
```
Comment (fyrnaga) [inline source/includes/stackpath/_network_policy_rules.md:12]: maybe add the workloadId parameter option information to filter for a particular workloadId

File: source/includes/stackpath/_network_policy_rules.md [hunk 2]
```diff
@@ -3,6 +3,114 @@
...
+      "source": "0.0.0.0/0",
+      "action": "inbound",
+      "protocol": "icmp",
+      "portRange": "",
+    },
+    {
+      "stackId": "bcc60174-50e6-44e4-bd45-463845124d87",
+      "workloadId": "6ca53aff-8930-46d6-ba86-afbeb0b46bb7",
+      "networkPolicyId": "2ac958f2-0976-4de9-95f6-3546fc10f8d0",
+      "id": "2ac958f2-0976-4de9-95f6-3546fc10f8d0/-812331665",
```
Comment (fyrnaga) [inline source/includes/stackpath/_network_policy_rules.md:36]: I will change the id structure in my pr

File: source/includes/stackpath/_network_policy_rules.md [hunk 3]
```diff
@@ -3,6 +3,114 @@
...
+------- | -----------
+`id`<br/>*string* | The ID of the network policy rule, in the form `workloadId/hashCode`.
+`stackId`<br/>*UUID* | The UUID of the stack to which the network policy belongs.
+`workloadId`<br/>*UUID* | The UUID of the workload to which the network policy rule is applied. Corresponds to the first workload ID in the network policy's list of instance selectors.
+`networkPolicyId`<br/>*UUID* | The UUID of the network policy to which the network policy rule belongs.
+`description`<br/>*string* | A summary of what this rule does or a name of this rule. It is highly recommended to give a unique description to easily identify a rule.
+`type`<br/>*string* | The type of network policy rule, either `inbound` or `outbound`.
+`source`<br/>*string* | A subnet that will define all the IPs allowed or denied by this rule.
+`action`<br/>*string* | The network policy rule action: `ALLOW` (allow traffic) or `BLOCK` (deny traffic).
+`protocol`<br/>*string* | Supported protocols are: `tcp`, `udp`, `tcpUdp`, `esp` and `ah`.
```
Comment (fyrnaga) [inline source/includes/stackpath/_network_policy_rules.md]: missing icmp and gre for the list and get

File: source/includes/stackpath/_network_policy_rules.md [hunk 4]
```diff
@@ -3,6 +3,114 @@
...
+      "portRange": "80",
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networkpolicyrules/</code>
```
Comment (y0geshdev) [inline source/includes/stackpath/_network_policy_rules.md]:
```suggestion
<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networkpolicyrules</code>
```

Comment (patrickspensieri) [reply source/includes/stackpath/_network_policy_rules.md]: Nice catch, fixed here https://github.com/cloudops/cloudmc-api-docs/pull/272/commits/92bbe2c9b998c05f83298c18a2328f888c41bc58

File: source/includes/stackpath/_network_policy_rules.md (hunk 3 above)
Comment (patrickspensieri) [reply source/includes/stackpath/_network_policy_rules.md]: Fixed here, shouldn't conflict with the edit rule PR https://github.com/cloudops/cloudmc-api-docs/pull/272/commits/91f423d22f68d3e4a32d05b96764d13e97ee1eeb

File: source/includes/stackpath/_network_policy_rules.md (hunk 1 above)
Comment (patrickspensieri) [reply source/includes/stackpath/_network_policy_rules.md:12]: Fixed here, shouldn't conflict with the edit rule PR https://github.com/cloudops/cloudmc-api-docs/pull/272/commits/91f423d22f68d3e4a32d05b96764d13e97ee1eeb

**Review by y0geshdev — APPROVED** [review]
Thanks for the changes .... lgtm.

**Review by fyrnaga — APPROVED** [review]
lgtm

**Pull request merged**